src/styles/utils/_vars.scss

```diff
@@ -1 +1 @@
-$c-gray: #eee;
+$green: #1a5a4c;
```

Comment:
Variable naming should be consistent with the naming convention. Consider using a more descriptive name like `$c-green` instead of `$green`.